src/types/index.ts

```diff
@@ -18,7 +18,21 @@ export type ThemeAppearance = "dark" | "light";
  * dashes instead of spaces. This value is used to determine which theme gets rendered
  * on the OpenResume site, and should be unique to each theme.
  */
-export type ThemeName = "default"; // Add more themes here, e.g. "default" | "my-theme" | "another-theme"
+export type ThemeName = "default" | "davids-theme"; // Added new Davids theme
```

Comment:
Per: https://github.com/missionmike/openresume-theme/pull/11/files#r2108211799

```suggestion
export type ThemeName = "default" | "davids"; // // Add more themes here, e.g. "default" | "my-theme" | "another-theme"
```